File: PRACTICA PRIMER PARCIAL/Ejercicio19/Python/SerieSum.py
import math

x = 0.5 

sum = 0
numerator = 2
denominator = 2
sign = 1

for i in range(1, 11):
    term = (numerator * math.pow(2, i)) / math.factorial(denominator)
    sum += sign * term
    denominator += 2
    sign *= -1

print("Suma:", sum)

File: PRACTICA PRIMER PARCIAL/Ejercicio19/Java/SerieSum.java
public class SerieSum {
    public static void main(String[] args) {
        double x = 0.5;  // Valor de x

        double sum = 0;
        double numerator = 2;
        double denominator = 2;
        int sign = 1;

        for (int i = 1; i <= 10; i++) { 
            double term = (numerator * Math.pow(2, i)) / factorial(denominator);
            sum += sign * term;
            denominator += 2;
            sign *= -1;
        }

        System.out.println("Suma: " + sum);
    }

    public static double factorial(double n) {
        if (n <= 1) {
            return 1;
        } else {
            return n * factorial(n - 1);
        }
    }
}
